build/helm-package.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash 
+#     Copyright 2021 Dynatrace LLC
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+wget https://get.helm.sh/helm-v3.5.3-linux-amd64.tar.gz
```

Comment:
Did you research the option to use package manager instead?

Reply:
I researched installation with script (described here: https://helm.sh/docs/intro/install/) but it wasn't up-to-date (I mean helm v.3) and when I saw "is generally up to date" in  package manager section ("From Apt (Debian/Ubuntu)") I decided to use wget. 

But I checked now and installation with package manager is also working properly - should I change the approach of installation?

Reply:
Package managers are generally safer and more robust than fetching with script, but if this is non-official repository, then you are right we probably should stick with script.